src/simulator_core/entities/utility/influxdb_reader.py

```diff
@@ -0,0 +1,123 @@
+"""Module to read the esdl profiles from an energy system."""
+from esdl.profiles.influxdbprofilemanager import InfluxDBProfileManager
+from esdl.units.conversion import ENERGY_IN_J, POWER_IN_W, convert_to_unit
+import esdl
+from esdl.esdl_handler import EnergySystemHandler
+from typing import Dict
+import pandas as pd
+
+
+def parse_esdl_profiles(esh: EnergySystemHandler) -> Dict[str, pd.DataFrame]:
+    """Method to parse the esdl profiles from an energy system.
+
+    This method tries to parse the esdl profiles from an energy system.
+    :param esh: EnergySystemHandler with the energy system
+    :param start_date: datetime.datetime with the start date
+    :param end_date: datetime.datetime with the end date
+    :return: dict with the parsed profiles
+    """
+    data_set = {}
+    for profile in esh.get_all_instances_of_type(esdl.InfluxDBProfile):
+        data_set[profile.id] = get_data_from_profile(profile)
+    return data_set
+
+
+def get_data_from_profile(profile: esdl.InfluxDBProfile) -> pd.DataFrame:
+    """Method to get the data from the esdl influxdb profile.
+
+    This method tries to get the data from the esdl profile.
+    :param esdl_profile: esdl.Profile with the profile
+    :return: pandas.DataFrame with the data
+    """
+    influx_cred_map = {
```

Comment:
Maybe discuss with Michiel if this is the best method to communicate with this database?

Reply:
Nee we moeten credentials op halen uit enviroment variabele. Dus pas ik nog aan